pkg/version/version.go

```diff
@@ -31,7 +31,7 @@ var (
 	State       string
 
 	// MaxPluginVersion must be increased to add new plugin and make sure the new plugin version is same to this value
-	MaxPluginVersion = 3
+	MaxPluginVersion = 4
```

Comment:
Not sure how we previously missed it, but this file needs 2017 added to copyright header.